README.md

```diff
@@ -1,18 +1,39 @@
 # Networking
 
-[![Coverage](https://img.shields.io/badge/Coverage-100%25-darkgreen?style=flat-square)]
-[![Platforms](https://img.shields.io/badge/Platforms-iOS_iPadOS_macOS_watchOS-lightgrey?style=flat-square)]
-[![Swift](https://img.shields.io/badge/Swift-5.9+-blue?style=flat-square)]
-
-A networking layer using native `UrlSession` and Swift concurrency.
-
-## Requirements
+![Coverage](https://img.shields.io/badge/Coverage-100%25-darkgreen?style=flat-square)
+![Platforms](https://img.shields.io/badge/Platforms-iOS_iPadOS_macOS_watchOS-lightgrey?style=flat-square)
+![Swift](https://img.shields.io/badge/Swift-5.9+-blue?style=flat-square)
+
+A networking layer using native `URLSession` and Swift concurrency.
+
+- [Requirements](#requirements)
+- [Installation](#installation)
+- [Overview](#overview)
+- [Basics](#basics)
+    - [Making requests](#makingRequests)
+    - [Downloading files](#downloadingFiles)
+    - [Uploading files](#uploadingFiles)
+    - [Request authorization](#requestAuthorization)
+- [Requestable](#requestable)
+- [APIManager](#apiManager)
+- [DownloadAPIManager](#downloadManager)
+- [UploadAPIManager](#uploadManager)
+- [Modifiers](#modifiers)
+- [Interceptors](#interceptors)
+    - [LoggingInterceptor](#loggingInterceptor)
+    - [AuthorizationTokenInterceptor](#authorizationTokenInterceptor)
+- [Processors](#processors)
+    - [StatusCodeProcessor](#statusCodeProcessor)
+    - [EndpointRequestStorageProcessor](#endpointRequestStorageProcessor)
+- [Associated array query parameters](#arrayQueryParameters)
+
+## Requirements <a name="requirements"></a>
```

Comment:
Doesn't it work automatically without doing those anchors? 🤔 Because it should 😄

Reply:
Good point, done ✅